DH_Engine/Classes/DHPlayer.uc

```diff
@@ -64,6 +64,7 @@ var     float                   DHSwayDampingFactor;
 var     float                   DHStandardTurnSpeedFactor;
 var     float                   DHHalfTurnSpeedFactor;
 var     globalconfig float      DHISTurnSpeedFactor;        // 0.0 to 1.0
+var     globalconfig float      DHISBipodTurnSpeedFactor;     // 0.0 to 1.0
```

Comment:
This has no default value set, so it will be 0, thus making it impossible to move the view around by default. You should put upgrade path logic that defaults the bipod turn speed factor to be equal to the ironsight turn speed factor. The upgrade logic usually goes in `DHMainMenu`. Perhaps that logic could be moved to it's own class as well (i.e., `DHVersionMigration` or something)

Reply:
I've added DHBipodTurnSpeedFactor=1.0 to the defaultproperties to DHPlayer.
So it won't affect the players until they change it in the settings.